web/src/main/java/com/bazaarvoice/emodb/web/scanner/config/ScheduledScanConfiguration.java

```diff
@@ -62,6 +64,19 @@
     @JsonProperty ("requestRequired")
     private boolean _requestRequired = false;
 
+    // Flag to indicate the maximum number of rows to scan in a single range scan task.
+    @Valid
+    @NotNull
+    @JsonProperty ("rangeScanSplitSize")
+    private int _rangeScanSplitSize = 1000000;
+
+    // Flag to indicate the maximum time a range scan can run before it is automatically stopped and remaining work
+    // split to a new task
+    @Valid
+    @NotNull
+    @JsonProperty ("maxRangeScanTime")
+    private String _maxRangeScanTime = "PT10M";
```

Comment:
Because of the Joda Module this shouldn't be necessary.  Try making this type a simple `Duration`; that should work.